.circleci/config.yml

```diff
@@ -23,15 +23,16 @@ jobs:
           name: publish to GCP container registry
           command: |
             GIT_COMMIT_SHA=$(git rev-parse HEAD)
+            TEST_RUNNER_TAG=gcr.io/omisego-development/omg-js-testrunner:omg-js-testrunner-$GIT_COMMIT_SHA
```

Comment:
This can be

`TEST_RUNNER_TAG=gcr.io/omisego-development/omg-js-testrunner:$GIT_COMMIT_SHA`

Reply:
The tag would be sha only in this case. Wouldn’t that be slightly more unreadable? Or that is more of the common way to tag?